osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -291,6 +291,7 @@ protected override void Update()
         {
             base.Update();
             detailsArea.Height = wedgesContainer.DrawHeight - titleWedge.LayoutSize.Y - 4;
+            filterControl.InformationalNote = $@"{carousel.MatchedBeatmapsCount} matches";
```

Comment:
Is there a reason the old code (which handled plurals correctly) wasn't copied?

Reply:
Ah I glossed over that part, I'll add it.